assets/airports/kmht.json

```diff
@@ -0,0 +1,976 @@
+        "OLDMN":["N42d48m11.090", "W71d30m47.400"],
+        "ONGEE":["N43d00m04.590", "W71d21m29.410"],
+        "ORDIE":["N43d00m10.220", "W71d28m32.590"],
+        "PELAN":["N42d45m20.000", "W71d19m47.410"],
+        "PNARD":["N42d49m47.460", "W71d55m40.440"],
+        "PONCT":["N42d44m48.830", "W73d48m48.070"],
+        "PPORT":["N43d05m23.590", "W70d59m29.080"],
+        "PURBL":["N42d57m44.520", "W71d40m38.570"],
+        "PUTRE":["N43d07m56.630", "W71d20m54.780"],
+        "QUINZ":["N42d46m23.520", "W73d03m00.520"],
+        "RBELA":["N42d49m57.990", "W70d57m17.960"],
+        "RKA":  ["N42d27m58.790", "W75d14m21.221", "Rockdale"],
+        "ROZZE":["N42d49m45.260", "W71d59m32.850"],
+        "SHOEL":["N42d59m27.000", "W70d52m40.390"],
+        "SHOWZ":["N42d48m19.530", "W71d34m51.650"],
+        "SITLE":["N42d57m55.390", "W71d27m12.740"],
+        "SMYTH":["N42d30m16.560", "W73d53m05.860"],
+        "SWEDE":["N42d28m40.120", "W74d40m08.110"],
+        "TEETO":["N42d48m47.150", "W71d42m27.410"],
+        "TYLIR":["N43d03m32.930", "W71d10m01.620"],
+        "WAVBO":["N43d03m33.726", "W71d16m39.454"],
+        "WOBLY":["N42d45m30.080", "W73d29m55.810"],
+        "XOHWI":["N42d53m29.660", "W71d24m35.710"],
+        "YANTC":["N41d33m22.810", "W71d59m56.950"],
+        "YOULL":["N42d48m47.080", "W71d21m49.170"]
+    },
+    "runways":[
+        {
+            "name": ["6", "24"],
+            "end": [
+                ["N42d55m40.139", "W71d26m58.814", "221ft"],
+                ["N42d56m37.040", "W71d25m47.849", "242ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["17", "35"],
+            "end": [
+                ["N42d56m32.190", "W71d26m23.350", "216ft"],
+                ["N42d55m04.670", "W71d25m31.837", "266ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "MHT1": {
+            "icao": "MHT1",
+            "name": "Manchester One",
+            "rwy": {
+                "KMHT6" : ["#039"],
+                "KMHT24": ["#237"],
+                "KMHT17": ["#172"],
+                "KMHT35": ["#352"]
+            },
+            "body": [],
+            "exitPoints": {
+                "BOS": ["BOS"],
+                "CAM": ["CAM"],
+                "CON": ["CON"],
+                "ENE": ["ENE"],
+                "GDM": ["GDM"]
+            },
+            "draw": [
+                ["BOS*"],
+                ["CAM*"],
+                ["CON*"],
+                ["ENE*"],
+                ["GDM*"]
+            ]
+        },
+        "PPORT4": {
+            "icao": "PPORT4",
+            "name": "Port Four",
+            "rwy": {
+                "KMHT6" : [["TYLIR", "A30+"]],
+                "KMHT24": [["TYLIR", "A30+"]],
+                "KMHT17": [["TYLIR", "A30+"]],
+                "KMHT35": [["TYLIR", "A30+"]]
+            },
+            "body": ["TYLIR", "PPORT", ["SHOEL", "A50+"], ["RBELA", "A68+"], ["COTEE", "A93+"]],
```

Comment:
NELIE and CCC transitions skip over COTEE, so it should be moved to the BAF exitPoint definition.

There are no altitude restrictions on this SID other than at TYLIR. The altitude restrictions are MSAs so shouldn't be defined as part of the SID.

Reply:
Thank you, I really didnt read that part of the chart.

Reply:
✔️